packages/investlab-domain/src/domain.module.ts

```
@@ -0,0 +1,6 @@
+import { ContainerModule } from "inversify";
+
+export const domainModule = new ContainerModule(({}) => {
```

ESLint

[error] Unexpected empty object pattern.

(no-empty-pattern)